easy_biologic/program.py

```diff
@@ -73,6 +73,27 @@
 from .lib import ec_lib as ecl
 from .lib import data_parser as dp
 
+# Try to import pandas for faster file writing
+try:

```

Comment:
I would 
1. Use a `try ... except ... else` block here, moving all the `pandas` setup code into the `else` block.
2. Change `pandas_installed` to `_pandas_installed` or `PANDAS_INSTALLED` (or something similar) to differentiate it from the typical local variables. Same for `pd_csv_kwargs`.

It may also be worth adding `pandas` as an extra.
(See [this SO answer](https://stackoverflow.com/a/27361558/2961550).)